gradle/libs.versions.toml

```diff
@@ -40,10 +38,9 @@ detekt_formatting = { module = "io.gitlab.arturbosch.detekt:detekt-formatting",
 espresso_core = { module = "androidx.test.espresso:espresso-core", version.ref = "espresso.core" }
 agp = { module = "com.android.tools.build:gradle", version.ref = "agp" }
 kgp = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
-dokka_gradle_plugin = { module = "org.jetbrains.dokka:dokka-gradle-plugin", version.ref = "dokka"}
-dokka_core = { module = "org.jetbrains.dokka:dokka-core", version.ref = "dokka"}
 
 [plugins]
 detekt = { id = "io.gitlab.arturbosch.detekt", version.ref = "detekt" }
 ktlint = { id = "org.jlleitschuh.gradle.ktlint", version.ref = "ktlint.gradle" }
 versions = { id = "com.github.ben-manes.versions", version.ref = "benmanesversion" }
+githooks = { id = "com.github.jakemarsden.git-hooks", version.ref = "githooks" }
```

Comment:
I'm not sure we want to add this plugin here as I don't believe this is a industry standard in the Android ecosystem

Reply:
Nope there is no standard way to do so

Reply:
Okay, I think including a pre-commit hook mechanism would be a good addition to this template, but I've removed it for now.